examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
I think the possibility to make more complex request is really nice.
But, I see that I could become pretty hard to read and to write, specially with all the nesting.
Another solution would be to switch to a single url check by input, it would allow us to flatten the configuration. I'm not a huge fan of this approach, as it would make the Logstash config longer and with a lot of redundancy.

I don't know what will be easier for the end user but if it was for me, I would keep the "complex" configuration